addons/web/static/src/views/form/form_controller.scss

```diff
@@ -848,11 +848,9 @@
         }
     }
 
-    // Boolean
-    .o_field_boolean {
-        margin-right: $o-form-spacing-unit;
-        max-width: 40px !important;
```

Comment:
As ugly as this rule looks... it's required in small screen to prevent the checkbox from going below the field's label.

Before:
![22367378-16-0-all runbot169 odoo com_web(Runbot Mobile)](https://user-images.githubusercontent.com/33146/210218690-d966c401-2667-4fc0-b456-4d9d5632288d.png)

After:
![22369060-16-0-all runbot131 odoo com_web(Runbot Mobile)](https://user-images.githubusercontent.com/33146/210218713-1c55a60e-bf5f-42c6-b0a6-176082dea499.png)

Reply:
Actually, same issue in larger screen (cf. Settings > Accounting, in the "AvaTax" section)

Reply:
Bringing the max-width: 40px back without the !important is the easiest solution and is working fine to go back to the previous behavior.

However I do prefer having input checkbox under their labels in mobile view. In my opinion it makes sense since it mimics the expected behavior of the other inputs. 

Furthermore, it makes it easier to reach for one hand phone holding (right handed users)
![image](https://user-images.githubusercontent.com/118886338/210322054-912e9c68-889c-4821-9033-5efdb1a3ddbd.png)

But indeed keeping it this way would mean some adaptation in the settings to cherry pick the impacted checkboxes but I'm up for it since I'm currently working on a very similar task: 3113372

Finally could you tell me where did you find the `receipt reminder` setting shown in your screenshot? I'd like to do some testings on this specific scenario but I struggle to find it.

Reply:
> Bringing the max-width: 40px back without the !important is the easiest solution and is working fine to go back to the previous behavior. (Edit: will fix for the settings. In form view the checkboxes were already under the labels before my fix)

Not only in the settings, but on small screens also. But yes, looks like the quick fix.

> However I do prefer having input checkbox under their labels in mobile view. In my opinion it makes sense since it mimics the expected behavior of the other inputs.

Well, not sure that I agree about that : from my point of view, it makes more sense to keep the checkbox and its label on the same line to emphasize the bond between those two elements (cf. https://m3.material.io/components/checkbox/guidelines). I invoke @stefanorigano on that matter :wink: 

> Furthermore, it makes it easier to reach for one hand phone holding (right handed users) ![image](https://user-images.githubusercontent.com/118886338/210322054-912e9c68-889c-4821-9033-5efdb1a3ddbd.png)
> 
> But indeed keeping it this way would mean some adaptation in the settings to cherry pick the impacted checkboxes but I'm up for it since I'm currently working on a very similar task: 3113372

Oooh ! Looking forward for this task :star_struck: as all those (miss-)spacing in the settings makes my eyes bleed everytime :+1: 

> Finally could you tell me where did you find the `receipt reminder` setting shown in your screenshot? I'd like to do some testings on this specific scenario but I struggle to find it.

My bad, forgot the "steps to reproduce".
As sent via Discord, here they are :
- Open Contacts app
- Open a contact
- Open the notebook's tab "Sales & Purchase"
- In the "Purchase" section, you'll find the "Receipt Reminder" checkbox

Reply:
>  from my point of view, it makes more sense to keep the checkbox and its label 

@pparidans , yes we agree. @mano-odoo is going further with the implementation btw, we should be close.
He'll ping us when it's done 👍

Reply:
@stefanorigano @pparidans it's (finally) up for review.